src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -5178,6 +5178,21 @@ def get_disable_cost_analysis(self) -> bool:
         # because it's already checked in _get_enable_cost_analysis
         return self.raw_param.get("disable_cost_analysis")
 
+    def get_if_match(self) -> Union[str, None]:
```

Comment:
don't forget to use it in put_mc (both create and update)

Reply:
Added